tests/testsuite/cargo_config.rs

```diff
@@ -273,7 +273,7 @@ target.\"cfg(target_os = \\\"linux\\\")\".runner = \"runme\" # [ROOT]/home/.carg
 
     cargo_process("config get --show-origin build.rustflags -Zunstable-options")
         .cwd(&sub_folder.parent().unwrap())
-        .masquerade_as_nightly_cargo()
+        .masquerade_as_nightly_cargo(&["cargo config"])
```

Comment:
Eh, guess its good enough
- `unstable-options` is an umbrella feature
- nothing else really to specify

Reply:
[`cargo config`](https://github.com/rust-lang/cargo/blob/d583b21e1494268be2cc8c19950e94f03ae3bce2/src/doc/src/reference/unstable.md#cargo-config) is unstable which is why I used it. The question I have is should we prefer dashes? as in `cargo-config`? It might be slightly better